Publishing sourceforge news failed

```
 curl: (3) URL using bad/illegal format or missing URL
  Error: Process completed with exit code 3.
```

The corresponding code:

```
        pmd_ci_sourceforge_createDraftBlogPost "${release_name} released" "${rendered_release_notes_with_links}" "pmd,release"
        SF_BLOG_URL="${RESULT}"

# ....

pmd_ci_sourceforge_publishBlogPost "$SF_BLOG_URL"
```

--- a/README.md
+++ b/README.md
@@ -300,16 +300,17 @@ bash -c 'set -e; \
          pmd_ci_sourceforge_createDraftBlogPost "draft post 2" "text without labels" ; \
          blog="${RESULT}" ; \
          echo "URL: ${blog}" ; \
+         #pmd_ci_sourceforge_publishBlogPost "${blog}" ; \
          ' $(pwd)/test.sh
 ```
 
 Note that "pmd_ci_sourceforge_selectDefault" won't be successful, because the file to be selected as default
 doesn't exist.
 
-Don't forget to delete https://sourceforge.net/projects/pmd/files/pmd/Release-Script-Test and
-https://pmd.sourceforge.io/test-Release-Script-Test after the test.
+Don't forget to delete <https://sourceforge.net/projects/pmd/files/pmd/Release-Script-Test> and
+<https://pmd.sourceforge.io/test-Release-Script-Test> after the test.
 
-And also the created blog posts under https://sourceforge.net/p/pmd/news/.
+And also the created blog posts under <https://sourceforge.net/p/pmd/news/>.
 
 #### inc/maven.bash
 

--- a/scripts/inc/sourceforge-api.bash
+++ b/scripts/inc/sourceforge-api.bash
@@ -185,7 +185,7 @@ function pmd_ci_sourceforge_createDraftBlogPost() {
       --form "state=draft" \
       --form "text=$text" \
       --form "title=$title" \
-      https://sourceforge.net/rest/p/pmd/news | grep -i "location: "|cut -d " " -f 2)
+      https://sourceforge.net/rest/p/pmd/news | grep -i "location: "|cut -d " " -f 2|tr -d "\r\n")
 
     pmd_ci_log_success "Created sourceforge blog post: ${RESULT}"
 }
@@ -199,7 +199,8 @@ function pmd_ci_sourceforge_publishBlogPost() {
     local url="$1"
 
     local response
-    response=$(curl --request POST --header "Authorization: Bearer ${PMD_SF_BEARER_TOKEN}" \
+    response=$(curl --silent --request POST \
+      --header "Authorization: Bearer ${PMD_SF_BEARER_TOKEN}" \
       --form "state=published" \
       "${url}")
     pmd_ci_log_debug "Response: ${response}"

